cli/flox/doc/flox-activate.md

```diff
@@ -93,6 +93,11 @@ See [`manifest.toml(5)`](./manifest.toml.md) for more details on shell hooks.
 
 ## Variables set by `flox activate`
 
+`$FLOX_ENV`
+:   Contains the path to the built environment. This directory contains a merged
+    set of `bin`, `lib`, etc directories for all the packages in the
+    environment.
```

Comment:
nonblocking: I think it would be good to mention how this var can/should be used. E.g. you can use this in `scripts` and `profile` to refer to contents of your environment